README.md

```diff
@@ -70,11 +71,6 @@ const monitor: Monitor<DataType> = Monitors.builder({
           type: 'falling-edge',
           threshold: 0.5,
         },
-        {
```

Comment:
@tsmbl messageBuilder being deprecated? or just showing off the email / dialectThread here instead?